docker-compose.yml

```
@@ -0,0 +1,83 @@
+version: "3.4"
+services:
+  conference-app:
+    container_name: conference-app
+    env_file:
+      - .env
+    command:
+      - sh
+      - -c
+      - |
+        if [ -e tmp/pids/server.pid ]; then
+          rm tmp/pids/server.pid
+        fi
+        bundle install
+        bin/rails s -p 3000 -b 0.0.0.0
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - conference-app-vendor:/vendor
+    stdin_open: true
+    tty: true
+    depends_on:
+      - conference-app-yaichi
+      - conference-app-db
+      - conference-app-redis
+  conference-app-css:
+    extends:
+      service: conference-app
+    container_name: conference-app-css
+    command:
+      - sh
+      - -c
+      - |
+        bundle install
+        bin/rails tailwindcss:watch
+  conference-app-worker:
+    extends:
+      service: conference-app
+    container_name: conference-app-worker
+    command:
+      - sh
+      - -c
+      - |
+        if [ -e tmp/pids/server.pid ]; then
+          rm tmp/pids/server.pid
+        fi
+        bundle install
+        bundle exec sidekiq --queue default
+    depends_on:
+      - conference-app-yaichi
+      - conference-app-db
+      - conference-app-redis
+    stdin_open: true
+    tty: true
+  conference-app-db:
+    container_name: conference-app-db
+    image: postgres:14.4-alpine
```

We use `postgres:15.4-bookworm` in CI (see `.github/workflows/test.yml`). It's good to use same image in development and test.

```suggestion
    image: postgres:15.4-bookworm
```

---

Thank you! Added the change with commit 6180db8